build-system/tasks/get-zindex/index.js

```diff
@@ -52,17 +54,16 @@ const logChecking = (filename) =>
 const sortedByEntryKey = (a, b) => a[0].localeCompare(b[0]);
 
 /**
- * @param {!Object<string, !Array<number>} acc accumulator object for selectors
- * @param {!Rules} css post css rules object
+ * @param {!Object<string, string>} acc accumulator object for selectors
+ * @param {!Postcss.Rule} css post css rules object
  */
 function zIndexCollector(acc, css) {
   css.walkRules((rule) => {
     rule.walkDecls((decl) => {
       // Split out multi selector rules
-      let selectorNames = rule.selector.replace('\n', '');
-      selectorNames = selectorNames.split(',');
```

Comment:
Could also just merge these two directly in the const: `const selectorNames = rule.selector.replace('\n', '').split(',');`

Reply:
I think this is preferable to moving everything into the `if` block like you've done.